po/sk.popie

```diff
@@ -40,6 +40,9 @@ msgstr Emoji bez karma hodnoty
 msgid All server emojis have been assigned a karma value.
 msgstr Všetkým emoji na tomto serveri je pridelená hodnota.
 
+msgid That emoji is not from this server
+msgstr
```

Comment:
Can you add translation for this string? You can use Google Translator, it would be weird to leave single line untranslated in fully localised module.